react/features/authentication/components/web/LoginDialog.js

```diff
@@ -10,8 +10,8 @@ import { Dialog } from '../../../base/dialog';
 import { translate, translateToHTML } from '../../../base/i18n';
 import { JitsiConnectionErrors } from '../../../base/lib-jitsi-meet';
 import { connect as reduxConnect } from '../../../base/redux';
-import { authenticateAndUpgradeRole } from '../../actions.native';
-import { cancelLogin, hideLoginDialog } from '../../actions.web';
+import { authenticateAndUpgradeRole } from '../../actions.any';
```

Comment:
actions.web and actions.native should re-export everything from actions.any, so you only import from one of them.

Reply:
Done.